src/Features/Core/Portable/GoToDefinition/AbstractGoToDefinitionSymbolService.cs

```diff
@@ -72,7 +73,8 @@ public async Task<(int? targetPosition, TextSpan tokenSpan)> GetTargetIfControlF
         if (token == default)
             return default;
 
-        var semanticModel = await document.GetRequiredSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+        // We don't need nullable information to compute control flow targets.  So avoid expensive work computing this.
+        var semanticModel = await document.GetRequiredNullableDisabledSemanticModelAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
debugged through, and there are no other codepaths where we retrieve semantic models for this feature.